content/docs/gto/user-guide/dvc.md

```diff
@@ -0,0 +1,144 @@
+# GTO with DVC
+
+In many ML projects, data isn't stored in a Git repository and needs to be
+downloaded from external sources. [DVC](https://dvc.org) is a common way to
+store binaries for the artifacts registered with GTO.
+
+<details>
+
+### Learn about different approaches to this
+
+1. You can commit artifacts to Git repo. If they aren't small enough, this is
+   not recommended. To bypass this limitation, you can use
+   [Git-lfs](https://git-lfs.github.com).
+2. You can version binaries with [DVC](https://dvc.org/) and commit pointers to
+   them to the repo. This is the recommended approach for large files.
+3. You can version binaries manually somewhere, specifying URL to them as `path`
+   in `artifacts.yaml`. This can be done, if your artifacts are already
+   versioned by some external system.
+
+</details>
+
+If you are new to DVC, you should start with
+[DVC Get Started](https://dvc.org/doc/start) first, and then come back to this
+Guide.
+
+<!-- ```
+
+dvc init --no-scm dvc remote add az azure://container-name export
+AZURE_STORAGE_CONNECTION_STRING='YOUR_CONNECTION_STRING'
+
+dvc import-url --no-download azure://container-name/data.parquet
+
+``` -->
+
+## Tracking an artifact with DVC
+
+First, we need to start tracking artifact with DVC. If you produce this artifact
+in DVC Pipelines, it's done automatically.
+
+If the artifact is located inside your Git repo, you can use `dvc add`:
+
+```cli
+$ dvc add model.pkl
+$ git add model.pkl.dvc
+```
+
+If the artifact is located in some external storage, we can use `dvc import-url`
+to still keep metainformation about it in the repo (use `--no-download` to skip
+downloading it):
+
+```cli
+$ dvc import-url --no-download s3://container/model.pkl
+$ git add model.pkl.dvc
+```
+
+## Annotating DVC-tracked artifacts
+
+Once the artifact is tracked with DVC within your repo, we can annotate it with
+GTO:
+
+```cli
+$ gto annotate model --path model.pkl
+```
+
+This will modify `artifacts.yaml`, adding:
+
+```yaml
+model:
+  path: model.pkl
+```
+
+Now you should commit changes to Git, and you can register versions and assign
+stages referencing the new commit.
+
+```cli
+$ git add artifacts.yaml
+$ git commit -m "version artifact binaries with DVC and annotate it with GTO"
+$ git push
+```
+
+Now your changes is live in your Git repo and you can download your artifact to
+use it 🙌
+
+## Downloading artifacts
+
+When you want to download GTO artifact which binaries are stored with DVC, you
+need to use `dvc get` or `dvc import` command:
```

Comment:
Copy edit
```suggestion
To download GTO artifact files tracked with DVC, you
can use the `dvc get` or `dvc import` commands.
```
But...

Reply:
Wait, what about `dvc pull`? Seems like we're missing that scenario (which is the simplest). Maybe cover that and only mention+link to the `dvc import/get` refs for more advanced uses.

Reply:
To use `dvc pull`, you need to be inside of the repo. In many cases, you just will want to get the artifact without cloning the repo. But I've added `dvc pull` mention, thanks for the suggestion.

Reply:
Thanks